Source/WTF/wtf/BitSet.h

```diff
@@ -157,6 +159,9 @@ class BitSet final {
     // and then casted to unsigned, meaning that set(31) when WordType is
     // a 64 bit unsigned int would give 0xffff8000
     static constexpr WordType one = 1;
+    static constexpr bool useSIMD = words >= 64;
+    using Vec = SIMD::VectorType<WordType>;
+    static constexpr size_t stride = SIMD::stride<WordType>;
```

Comment:
i haven't fully formalized my thought. we can perhaps house SIMD-related configuration at compile-time like this:

```cpp
template<auto words, typename WordType>
struct SIMDConfig {
    static constexpr bool shouldUseSIMD = words >= 64;
    static constexpr size_t stride = shouldUseSIMD ? SIMD::stride<WordType> : 0;
};
```

`using config = SIMDConfig<words, WordType>;`

and utilize it when branching:

`if constexpr (config::shouldUseSIMD) {}`

if you want read stride value:

`for (; idx + config::stride <= words; idx += config::stride) {}`